e2e/credentials-server.e2e.spec.ts

```diff
@@ -251,8 +251,52 @@ describeWithCreds('Credentials server', () => {
 
     ({ stdout } = await rdctlCredWithStdin('get', bobsURL));
     expect(stdout).toContain('credentials not found in native keychain');
+  });
 
-    // Don't bother trying to test erasing a non-existent credential, because the
-    // behavior is all over the place. Fails with osxkeychain, succeeds with wincred.
+  test('complains when the limit is exceeded (on the sever)', async() => {
+    const args = [
+      'shell',
+      'bash',
+      '-c',
+      `source /etc/rancher/desktop/credfwd; \
+       DATA="@-"; \
+       SECRET=$(tr -dc 'A-Za-z0-9,._=' < /dev/urandom |  head -c5242880); \
+       echo '{"ServerURL":"https://example.com/v1","Username":"bob","Secret":"'$SECRET'"}' |
+         curl --silent --show-error --user "$CREDFWD_AUTH" --data "$DATA" --noproxy '*' --fail-with-body "$CREDFWD_URL/store"
+      `
```

Comment:
There is no need to inline the `docker-credential-rancher-desktop` implementation; we can just call it:

```suggestion
      `SECRET=$(tr -dc 'A-Za-z0-9,._=' < /dev/urandom |  head -c5242880); \
       echo '{"ServerURL":"https://example.com/v1","Username":"bob","Secret":"'$SECRET'"}' |
         docker-credential-rancher-desktop store"
      `
```

Reply:
Same thing in the next test

Reply:
I had to change the output so it no longer discards output to `stderr`. Note that the output for the `golang` protocol is all over the place -- `List` returns `(string, error)`, `Get` returns `(string, string, error)`, and the others just return `error`.